packaging/installer-linux/installer-debian/src/main/resources/community/debian/neo4j.preinst

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+
+set -e
+
+NEO4J_LIB_DIR=/var/lib/neo4j
+NEO4J_DATA_DIR=/var/lib/neo4j/data
+
+OLD_PLUGINS_LINK=/var/lib/neo4j/plugins
+OLD_PLUGINS_DIR=/usr/share/neo4j/plugins
+OLD_IMPORT_DIR=/var/lib/data/import
+OLD_CERTIFICATES_DIR=/etc/neo4j/ssl
+
+NEW_DATABASES_DIR=/var/lib/neo4j/data/databases
+NEW_CERTIFICATES_DIR=/var/lib/neo4j/certificates
+NEW_PLUGINS_DIR=/var/lib/neo4j/plugins
+NEW_IMPORT_DIR=/var/lib/neo4j/import
+
+case "$1" in
+    upgrade)
+        dpkg-maintscript-helper rm_conffile "/etc/init.d/neo4j-service" -- "$@"
+        dpkg-maintscript-helper rm_conffile "/etc/neo4j/README.txt" -- "$@"
+        dpkg-maintscript-helper rm_conffile "/etc/neo4j/neo4j-http-logging.xml" -- "$@"
+        dpkg-maintscript-helper rm_conffile "/etc/neo4j/neo4j-server.properties" -- "$@"
+        dpkg-maintscript-helper rm_conffile "/etc/neo4j/neo4j.properties" -- "$@"
+
+        if [ -d "${OLD_CERTIFICATES_DIR}" ]; then
+            mkdir -p "${NEW_CERTIFICATES_DIR}" && chown neo4j.nogroup "${NEW_CERTIFICATES_DIR}" && chmod 0755 "${NEW_CERTIFICATES_DIR}"
```

Comment:
I don't know what the difference is. `:` is the only way it's documented, but both work. Weird.